.ci/install_ovmf_sev.sh

```diff
@@ -19,8 +19,8 @@ main() {
 	clone_katacontainers_repo
 
 	pushd $katacontainers_repo_dir
-	sudo -E PATH=$PATH bash ${buildscript} --build=cc-sev-ovmf
```

Comment:
I think I mentioned to @UnmeshDeodhar that this script should be using `build_static_artifact_and_install` like all the others. This is the kind of issue that arises when not following the "standard".